assets/src/blocks/module-navigation/block.json

```diff
@@ -24,7 +24,13 @@
         "fontFamily": true,
         "textTransform": true
       },
-      "shadow": true
+      "shadow": true,
+      "__experimentalBorder": {
```

Comment:
This isn't appearing in the editor. Can you please  think about what might be going wrong?

Reply:
This doesn't make it seem promising https://github.com/WordPress/gutenberg/discussions/69756 but I have to guess that there's *some* way to turn it on. Maybe it's theme specific? https://developer.wordpress.org/themes/global-settings-and-styles/settings/border/ If so, perhaps you can let me know in a comment.